### PR TITLE
Dgus update

### PR DESCRIPTION
**Pull request by GerogeFu**
Fix DGUS bugs and update DUGS FYSETC definition

### CODE REVIEW

File: Marlin/Configuration_adv.h [hunk 1]
```diff
@@ -1284,11 +1284,12 @@
 // Additional options for DGUS / DWIN displays
 //
 #if HAS_DGUS_LCD
-  #define DGUS_SERIAL_PORT 2
+  #define DGUS_SERIAL_PORT 3
```
Comment (thinkyhead) [inline Marlin/Configuration_adv.h:1291]: DGUS might be attached to any board at all. Why is '3' is any better than '2'?

File: Marlin/Configuration_adv.h [hunk 2]
```diff
@@ -1284,11 +1284,12 @@
...
   #define DGUS_BAUDRATE 115200
 
   #define DGUS_RX_BUFFER_SIZE 128
   #define DGUS_TX_BUFFER_SIZE 48
+  #define SERIAL_TX_BUFFER_SIZE 256
```
Comment (thinkyhead) [inline Marlin/Configuration_adv.h]: This define is not used anywhere in Marlin. There is a `TX_BUFFER_SIZE`, however. It's defined later in this same file.

Comment (GerogeFu) [reply Marlin/Configuration_adv.h]: This is used in ststm32 framework.

Comment (thinkyhead) [reply Marlin/Configuration_adv.h]: But defining a same-named thing here has no influence on the STM32 framework. Its .CPP files cannot see it.

**Comment by GerogeFu** [comment]
I waiting for this merging. @thinkyhead

**Comment by thinkyhead** [comment]
> I waiting for this merging.

Some fixes are needed. See the comments I made on the code above.

1. DGUS serial port. Why is 3 a better default than 2? Which boards are you considering?
2. The added `SERIAL_TX_BUFFER_SIZE` define is pointless. It needs to be removed.

---
I set `DGUS_RX_BUFFER_SIZE` to 256 just in case that is the real intent.

File: Marlin/Configuration_adv.h (hunk 1 above)
Comment (GerogeFu) [reply Marlin/Configuration_adv.h:1291]: I am debuging FYSETC S6 , it use port 3. And it supports most for DGUS screen , at the moment , in the market. So i change it to 3.

File: Marlin/Configuration_adv.h (hunk 2 above)
Comment (GerogeFu) [reply Marlin/Configuration_adv.h]: My mistake. Delete it.

**Comment by GerogeFu** [comment]
And don't change DGUS_RX_BUFFER_SIZE to 256 as 8 bit MCU may don't have enough RAM for it.

**Comment by thinkyhead** [comment]
> 8 bit MCU may don't have enough RAM for it.

It's good to know I'm not the only one who still cares about the 644p people….

**Pull request merged**